src/backend/joanie/payment/backends/dummy.py

```diff
@@ -127,7 +128,11 @@ def _get_payment_data(
     ):
         """Build the generic payment object."""
         order_id = str(order.id)
-        payment_id = self.get_payment_id(order_id)
+        payment_id = (
+            self.get_payment_id(installment.get("id"))
+            if installment
+            else self.get_payment_id(order_id)
+        )
```

Comment:
@kernicPanel introduces installment requirement. We now always have an installment in a payment action. So you can change this to use only the `installement` id. See #847